src/AutoRest.CSharp/Common/AutoRest/Plugins/CSharpGen.cs

```diff
@@ -54,6 +56,40 @@ public async Task<GeneratedCodeWorkspace> ExecuteAsync(CodeModel codeModel)
             return project;
         }
 
+        private async Task<CSharpCompilation?> LoadBaselineContract()
+        {
+            if (!Configuration.AzureArm)
```

Comment:
let us add a TODO item here since this is temporary.

Reply:
Added a comment, not a TODO, think we are far away from enabling it for HLC.